launcher/minecraft/auth/steps/OfflineStep.cpp

```diff
@@ -0,0 +1,18 @@
+#include "OfflineStep.h"
+
+#include "Application.h"
+
+OfflineStep::OfflineStep(AccountData* data) : AuthStep(data) {};
```

Comment:
The semicolon after the closing curly bracket causes a build failure with -Werror=pedantic.

Reply:
Fixed, thanks!